ros_buildfarm/ci_job.py

```diff
@@ -195,6 +195,13 @@ def configure_ci_job(
         underlay_source_paths = (underlay_source_paths or []) + \
             ['$UNDERLAY_JOB_SPACE']
 
+    trigger_jobs = None
+    if build_file.jenkins_job_upstream_trigger:
```

Comment:
Since the default for the config option is an empty list both lines 198 and 199 are unnecessary.